### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by ParagXD** [community/community#134409] [GitHub Education]
### Hi everyone,

I have claimed my developer pack in another account linked with the same email id and then I have to delete that account. After unlinkingy email I created this account and tried to claim pack but it says it has been already claimed.Now what should I do to access it??

**Comment by GithmiHashara** [comment]
Contact GitHub Support: Explain the situation and request a transfer of the developer pack to your new account.
Provide Details: Include details of the deleted account and your new account to help support verify your request.

Hope you'll get back your developer pack soon.

**Comment by davevad93** [comment]
Hi @ParagXD ,

Unfortunately, it's not possible to transfer the `Student Developer Pack` benefits from one account to another. Your benefits are tied to the email and account you deleted, so when you try to apply for the benefits with your new account it tells you that they have already been claimed.

You could try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and ask if it's possible to apply again with the same educational email, although I doubt it.

**Reply by ParagXD** [reply]
Ok Thank you for the suggestion
